assets/js/script.js

```diff
@@ -17,114 +19,142 @@ var prize = [
 var step = 0;
 var selection;
 var selectedQuestions = [];
-var correctAnswer;
-var totalQuestions = 547;
-var randomQuestion;
-var questionLength;
 var timeleft;
 var downloadTimer;
 var player;
+var chances = 3;
 
-var highScores = JSON.parse(localStorage.getItem('highScores')) || [];
-console.log(highScores);
+let highScores = JSON.parse(localStorage.getItem('highScores')) || [];
 
-// Function to generate unique set of 12 questions using Chance.js library
+/**
+ * Function to generate unique set of 12 questions using Chance.js library 
+ */
 
 function generateUnique() {
-    var uniques = chance.unique(chance.natural, 12, {
+    const uniques = chance.unique(chance.natural, 12, {
         min: 1,
         max: 547,
     });
     return uniques;
 }
 
-// Function to load questions from JSON file.
+/*
+ * Function to load questions from JSON file. 
+ */
+
 $(function() {
-    var xmlhttp = new XMLHttpRequest();
-    var url = 'questions.json';
+    const xmlhttp = new XMLHttpRequest();
+    const url = 'questions.json';
 
     xmlhttp.onreadystatechange = function() {
         if (this.readyState == 4 && this.status == 200) {
-            myArr = JSON.parse(this.responseText);
-            // selectQuestion(myArr);
-            loadQuestions();
-            $('#myModal').modal('hide');
+            let fullSetQuestions = JSON.parse(this.responseText);
+            loadQuestions(fullSetQuestions);
+            $('#startGameWindow').modal('hide');
             $('#game-screen').hide();
             $('#gameOverWindow').modal('hide');
             $('#timeOutWindow').modal('hide');
+            $('.answer-button').removeClass('wrong-answer correct-answer');
         }
+
     };
     xmlhttp.open('GET', url, true);
     xmlhttp.send();
-    $('.answer-button').removeClass('selected-answer correct-answer');
-});
 
-// Function to create sub array of 12 questions from big array using generated numbers
+});
 
-function loadQuestions() {
-    randomQuestion = generateUnique();
-    for (i = 0; i < 12; i++) {
-        var value = randomQuestion[i];
-        selectedQuestions.push(myArr[value]);
-        console.log(selectedQuestions[i]);
-    }
+/*
+ * Function to create sub array of 12 questions from big array using generated numbers 
+ */
 
+function loadQuestions(fullSet) {
+    const gameQuestionNumbers = generateUnique();
+    selectedQuestions = gameQuestionNumbers.map(numbers => fullSet[numbers]);
+    console.log(selectedQuestions);
```

Comment:
If you wanna leave this, state it in your readme so that the accessors know it's intentional